### ISSUE TITLE
Quiz App

### ISSUE THREAD

**Issue by aditi10701** [OyePradumya/Collection#55]
The above quiz app is built using react.js and node js. Kindly assign me this issue I want to work  under this issue 
#27 . ![image](https://user-images.githubusercontent.com/98349669/193466594-8d219646-7dd4-48d9-9036-225d4b97085d.png)

**Comment by OyePradumya** [comment]
Hey @aditi10701 if you are planning for Hacktoberfest then this will not get you qualification.

**Comment by aditi10701** [comment]
> Hey @aditi10701 if you are planning for Hacktoberfest then this will not get you qualification.

Can you please elaborate over this?
what sort of projects are accepted?

**Comment by OyePradumya** [comment]
> > Hey @aditi10701 if you are planning for Hacktoberfest then this will not get you qualification.
> 
> Can you please elaborate over this? what sort of projects are accepted?

Only PRs are accepted not issues. If you have code (full project only) or document (for any problem) then you may post in this repo which when merged gets you 1/4 complete.

**Comment by OyePradumya** [comment]
Go through the Guidelines.md file in this repo

**Comment by aditi10701** [comment]
I have the complete code for the same can I generate a pull request for the same?

**Comment by OyePradumya** [comment]
Yes. Make sure to include everything in a single folder that can be run without additional hustle.

**Comment by ig77ayush** [comment]
Please assign this issue to me
